packages/manager-react-components/package.json

```
@@ -55,8 +55,8 @@
     "@ovh-ux/manager-react-shell-client": "^0.8.5",
     "@ovh-ux/manager-tailwind-config": "^0.2.1",
     "@ovh-ux/manager-vite-config": "^0.9.0",
-    "@ovhcloud/ods-components": "^18.3.0",
-    "@ovhcloud/ods-themes": "^18.3.0",
+    "@ovhcloud/ods-components": "^18.5.0",
+    "@ovhcloud/ods-themes": "^18.5.0",
```

```suggestion
    "@ovhcloud/ods-components": "^18.5.1",
    "@ovhcloud/ods-themes": "^18.5.1",
```

And I think you need to push the yarn.lock changes

---

👍